service/src/main/kotlin/fi/espoo/evaka/application/ApplicationQueries.kt

```diff
@@ -222,9 +222,9 @@ fun Database.Read.fetchApplicationSummaries(
         if (distinctions.contains(ApplicationDistinctions.SECONDARY)) "f.preferredUnits && :units" else if (units.isNotEmpty()) "d.id = ANY(:units)" else null,
         if (authorizedUnitsForApplicationsWithoutAssistanceNeed != AclAuthorization.All) "((f.document->'careDetails'->>'assistanceNeeded')::boolean = true OR f.preferredUnits && :authorizedUnitsForApplicationsWithoutAssistanceNeed)" else null,
         if (authorizedUnitsForApplicationsWithAssistanceNeed != AclAuthorization.All) "((f.document->'careDetails'->>'assistanceNeeded')::boolean = false OR f.preferredUnits && :authorizedUnitsForApplicationsWithAssistanceNeed)" else null,
-        if ((periodStart != null || periodEnd != null) && dateType.contains(ApplicationDateType.DUE)) "daterange(:periodStart, :periodEnd, '[]') @> a.dueDate" else null,
-        if ((periodStart != null || periodEnd != null) && dateType.contains(ApplicationDateType.START)) "daterange(:periodStart, :periodEnd, '[]') @> (f.document ->> 'preferredStartDate')::date" else null,
-        if ((periodStart != null || periodEnd != null) && dateType.contains(ApplicationDateType.ARRIVAL)) "daterange(:periodStart, :periodEnd, '[]') @> a.sentdate" else null,
+        if ((periodStart != null || periodEnd != null) && dateType.contains(ApplicationDateType.DUE)) "includes(daterange(:periodStart, :periodEnd, '[]'), a.dueDate)" else null,
+        if ((periodStart != null || periodEnd != null) && dateType.contains(ApplicationDateType.START)) "includes(daterange(:periodStart, :periodEnd, '[]'), (f.document ->> 'preferredStartDate')::date)" else null,
+        if ((periodStart != null || periodEnd != null) && dateType.contains(ApplicationDateType.ARRIVAL)) "includes(daterange(:periodStart, :periodEnd, '[]'), a.sentdate)" else null,
```

Comment:
Would it make sense to also bind a FiniteDateRange here instead of `periodStart` and `periodEnd`? It seems to be so in all the other parts of this PR.

Reply:
Oh I see, both `periodStart` and `periodEnd` are nullable, so forget about my comment 😄